hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -74,11 +74,32 @@ def queries(query_group,
         salt '*' nebula.queries hour verbose=True
         salt '*' nebula.queries hour pillar_key=sec_osqueries
     '''
+    config = {}
+    if isinstance(query_file, list):
+        for file in query_file:
+            if 'salt://' in file:
+                file = __salt__['cp.cache_file'](file)
+            if os.path.isfile(file):
+                with open(file, 'r') as f:
+                    config = _dict_update(config,
+                                         yaml.safe_load(f),
+                                         recursive_update=True,
+                                         merge_lists=True)
+        if salt.utils.is_windows():
+            query_file = "salt://hubblestack_nebula/hubblestack_nebula_win_queries.yaml"
+        else:
+            query_file = 'salt://hubblestack_nebula/hubblestack_nebula_queries.yaml'
```

Comment:
this if/else statement isn't needed here, because if we want to run those files, they'll be in the list above and we will have already processed them